crates/sui-core/src/authority/authority_store.rs

```diff
@@ -1027,50 +1028,63 @@ impl AuthorityStore {
         &self,
         write_batch: &mut DBBatch,
         inner_temporary_store: InnerTemporaryStore,
+        effects: &TransactionEffects,
         _transaction: &VerifiedTransaction,
         _epoch_id: EpochId,
     ) -> SuiResult {
         let InnerTemporaryStore {
-            objects,
+            input_objects: objects,
```

Comment:
nit: This looks like it might be an artifact of an LSP renaming?  Should `objects` here also be renamed to `input_objects`?